doc/rtd/explanation/format.rst

```diff
@@ -121,6 +193,48 @@ The :command:`make-mime` subcommand takes pairs of (filename, "text/" mime
 subtype) separated by a colon (e.g., ``config.yaml:cloud-config``) and emits a
 MIME multipart message to :file:`stdout`.
 
+User data format to Mime Content type mapping
+---------------------------------------------
+
+We use the following format to outline the content-type the user is
+using
+
+.. code-block::
+
+   Content-Type: text/<content-type>
+
+Below is a mapping of a specific user data format to the `content-type`
```

Comment:
Not sure if this in an improvement but still i feel like this sentence is a little bit confusing.
```suggestion
Below is a mapping of each specific user data format to the `content-type`
```